filament/backend/src/DriverBase.h

```diff
@@ -176,7 +176,7 @@ class DriverBase : public Driver {
         CallbackData(CallbackData&&) = delete;
         CallbackData& operator=(CallbackData const &) = delete;
         CallbackData& operator=(CallbackData&&) = delete;
-        void* storage[8] = {};
+        void* storage[13] = {};
```

Comment:
you can't change that, it was very much chosen to be 64 bytes. there should have been a comment.